ProjFS.Mac/PrjFSKext/VirtualizationRoots.hpp

```diff
@@ -30,6 +30,9 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(
     vnode_t _Nonnull vnode,
     vfs_context_t _Nonnull context);
 
+VirtualizationRootHandle ActiveProvider_FindForPath(
```

Comment:
It will currently only find *active* roots. So even if the “from” path is in an offline root, it will return a “none” handle.

We can in principle re-instate tracking the path of offline roots but this will be even less reliable in the face of moves/renames than for active providers.